### PR TITLE
PO-10: Energy settings

### PR DESCRIPTION
**Pull request by itolegmysin**

### CODE REVIEW

File: src/pages/settings/energy/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import {Base} from 'resources/base';
+import {Refresher} from 'components/refresher';
+import {Toolbox} from 'components/toolbox';
+import {Logger} from 'components/logger';
+
+@inject()
+export class Energy extends Base {
+    constructor(...rest) {
+        super(...rest);
+        this.labelInputs = [];
```
Comment (khenderick) [inline src/pages/settings/energy/index.js]: An important remark here.

This page (pulse counters & energy module configuration) will be available on both the gateway as cloud builds. However, the label-related functionality/configuration is a cloud-only feature and must be hidden on gateway builds.

So;
* Label(input) related fields/wizards/configuration: Only to be shown in cloud builds
* Pulse counter & energy module configurations: Should work on both cloud and gateway builds.

**Comment by iOlivier** [comment]
As discussed in the call after the demo meeting:
- Sort by id
- Add edit button for every row (see attached screenshot)

Screenshot 2020-04-08 at 11 35 58

**Comment by iOlivier** [comment]
Don't forget to sort by id

**Comment by itolegmysin** [comment]
> Table views: sort by id

Done

**Pull request merged**